lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -156,7 +145,10 @@ test.serial('migrateGranuleRecord correctly migrates granule record', async (t)
     testGranule,
   } = t.context;
 
-  const [granuleCumulusId] = await migrateGranuleRecord(testGranule, knex);
+  const granuleCumulusId = await migrateGranuleRecord(testGranule, knex);
+  t.teardown(async () => {
```

Comment:
Mentioning here so I don't forget -- this pattern is interesting, although I'm used to seeing it revert spies/mocks,  but if I'm reading right, the order reverses in ava 4.... should we have `reverseTeardowns` enabled?

https://github.com/avajs/ava/blob/main/docs/02-execution-context.md

Reply:
In a lot of cases (almost all?) I only have one `teardown` per test, but generally yes we would probably want to enable `reverseTeardowns`

Also, guess who submitted that feature to Ava:

https://github.com/avajs/ava/pull/2505/files

Our very own @yjpa7145

Reply:
Fair enough - given the tests here, it's fine not to immediately sort it, but we should make sure we do.  I'll see about surfacing that.